website/meta/universe.json

```diff
@@ -1,5 +1,72 @@
 {
     "resources": [
+        {
+            "id": "scrubadub_spacy",
+            "title": "scrubadub_spacy",
+            "category": ["pipeline"],
+            "tags": [""],
+            "slogan": "Clean personally identifiable information from dirty dirty text using spaCy.",
+            "description": "scrubadub removes personally identifiable information from text. scrubadub_spacy is an extension that uses spaCy NLP models to remove personal information from text.",
+            "github": "LeapBeyond/scrubadub_spacy",
+            "pip": "scrubadub-spacy",
+            "url": "https://github.com/LeapBeyond/scrubadub_spacy",
+            "code_language": "python",
+            "author": "Thomas Bird",
+            "author_links": {
+                "twitter": "thomasbird",
+                "github": "thomasbird",
+                "website": "http://thomasbird.com"
+            },
+            "code_example": [
+                "import scrubadub, scrubadub_spacy",
+                "scrubber = scrubadub.Scrubber()",
+                "scrubber.add_detector(scrubadub_spacy.detectors.SpacyEntityDetector)",
+                "print(scrubber.clean(\"My name is Philip, I work at Explosion in Berlin, and my eMail is philip@explosion.ai btw. my super secret twitter login is username: philipvollet password: g-dragon180888\"))",
+                "# My name is {{NAME}}, I work at {{ORGANIZATION}} in {{LOCATION}}, and my eMail is {{EMAIL}} btw. my super secret twitter login is username: {{USERNAME}} password: {{PASSWORD}}"
+            ]
+        },
+        {
+            "id": "spacy-setfit-textcat",
+            "title": "spacy-setfit-textcat",
+            "category": ["research"],
+            "tags": ["SetFit", "Few-Shot"],
+            "slogan": "spaCy Project: Experiments with SetFit & Few-Shot Classification",
+            "description": "This project is an experiment with spaCy and few-shot text classification using SetFit",
+            "github": "pmbaumgartner/spacy-setfit-textcat",
+            "url": "https://github.com/LeapBeyond/scrubadub_spacy",
+            "code_language": "python",
+            "author": "Peter Baumgartner",
```

Comment:
If this fits better in the projects repo, I can do that.

Reply:
Up to you. It might get more traffic on the spaCy repo, but I think you created this before being part of the team, so if you'd prefer to keep in on your personal account, that'd be fine by me too!